frontend/src/components/PrescreeningForm.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import buttonStyle from "./Button.module.css";
+import styles from "./PrescreeningForm.module.css";
+import { Page } from "./index.ts";
+
+export function PrescreeningForm() {
+  // State to manage the selected values for each question
+  const [question1Value, setQuestion1Value] = useState("");
+  const [question2Value, setQuestion2Value] = useState("");
+  const [question3Value, setQuestion3Value] = useState("");
+
+  const pathOne = "/path1";
+  const pathTwo = "/path2";
+  const pathThree = "/path3";
+  const pathFour = "/path4";
+  const pathNQ = "/pathNQ";
+
+  const navigate = useNavigate();
+
+  const handleFormSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    let finalPath = "";
+
+    // Use the selected values to determine the finalPath
+    // Can modify this logic based on your requirements
+    if (question1Value === "op12" && question2Value === "op22" && question3Value === "op32") {
+      finalPath = pathOne;
+    } else if (
+      question1Value === "op11" &&
+      question2Value === "op22" &&
+      question3Value === "op32"
+    ) {
+      finalPath = pathTwo;
+    } else if (
+      question1Value === "op11" &&
+      question2Value === "op21" &&
+      question3Value === "op32"
+    ) {
+      finalPath = pathThree;
+    } else if (
+      question1Value === "op11" &&
+      question2Value === "op21" &&
+      question3Value === "op31"
+    ) {
+      finalPath = pathFour;
+    } else {
+      finalPath = pathNQ;
+    }
+
+    // setFinalPath(finalPath);
+
+    navigate(finalPath);
+  };
+
+  return (
+    <Page>
+      <h1 className={styles.title}>Prescreening Questions</h1>
+      <p className={styles.headingtwo}>
+        Please answer the following questions about your experience as an Interior Designer. Based
+        on your responses, we will automatically direct you into the appropriate pathway. View more
+        information about the pathways here.
+      </p>
+
+      <form onSubmit={handleFormSubmit} id="form1">
+        {/* Question 1 */}
+        <div>
```

Comment:
Each question is in div container. Label the div container and add more spacing between each question (marging top/bottom). Questions are a little too close to each other compared to Figma. 
<img width="822" alt="Screenshot 2024-04-15 at 12 36 54 PM" src="https://github.com/TritonSE/CCIDC-Application-Overhaul/assets/61575769/749692e7-2f66-4e36-8b9c-12423ada7041">